python/spacetime/store_server.py

```diff
@@ -90,6 +100,8 @@ def get(self, sim):
         ret["new"] = all_new
         ret["updated"] = all_updated
         ret["deleted"] = all_deleted
+
+        reset_timer(sim)
```

Comment:
That could still happen if the call takes too long. Maybe I should disable then reenable at the end.